bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/PredefinedTypes.java

```diff
@@ -185,6 +185,10 @@
     public static final RecordType XML_ITR_NEXT_RETURN_PI_TYPE =
             IteratorUtils.createIteratorNextReturnType(TYPE_PROCESSING_INSTRUCTION);
 
+    public static final Type ANY_AND_READONLY_TYPE =
+            ReadOnlyUtils.setImmutableTypeAndGetEffectiveType(new BUnionType(Arrays.asList(TYPE_READONLY, TYPE_ANY)));
```

Comment:
Why are we passing the union? Only any is enough, right?

```suggestion
            ReadOnlyUtils.setImmutableTypeAndGetEffectiveType(TYPE_ANY);
```

Reply:
Oh yes.